src/pageObject.js

```diff
@@ -1,16 +1,44 @@
 'use strict';
 
 class Header {
+  constructor(logo = 'Conduit logo') {
+    this.logo = logo;
+  }
 
-};
+  getLogo() {
+    return this.logo;
+  }
+}
 
 class PageObject {
+  constructor(url, header = new Header()) {
```

Comment:
It's better to avoid creating new objects inside default parameters. It's not a good practice because it can lead to unexpected behavior. Instead, you can check if `header` is provided in the constructor body and if not - create a new `Header` instance there.